xml/ay_bigfile.xml

```diff
@@ -6662,7 +6650,441 @@ config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Name of bonding device.
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Required for bond device. X is replaced by number from 0 e.g. <literal>bonding_slave0</literal> each slave have to have unique number.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          bonding_module_opts
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Options for bond device
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Used only with <literal>bond</literal> device.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          mtu
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Maximum transmission unit for given device
+         </para>
+        </entry>
+        <entry>
+         <para>
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          ethtool_options
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Ethtool options during device activation
+         </para>
+        </entry>
+        <entry>
+         <para>
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          zone
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Firewall zone name to which device is attached
+         </para>
+        </entry>
+        <entry>
+         <para>
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          vlan_id
+         </para>
+        </entry>
+        <entry>
+         <para>
+           ID used for this VLAN
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Used only with <literal>vlan</literal> device.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          etherdevice
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Device to which VLAN is attached.
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Used only with <literal>vlan</literal> device and required for it.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          bridge
+         </para>
+        </entry>
+        <entry>
+         <para>
+           "yes" if device is bridge
+         </para>
+        </entry>
+        <entry>
+         <para>
+           deprecated. Recognized by other attributes
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          bridge_ports
+         </para>
+        </entry>
+        <entry>
+         <para>
+           Comma separated list of bridge ports e.g. <literal>eth0 eth1</literal>.
```

Comment:
Hm, there's something fishy here, because both here and in the example above, the list is separated by spaces. So either the description or the examples need to be fixed.

Reply:
you are right. It is space separated list. My fault.

Reply:
No problem :).